pyaedt/modules/MaterialLib.py

```diff
@@ -66,7 +67,7 @@ def __len__(self):
         return len(self.material_keys)
 
     def __iter__(self):
-        return self.material_keys.itervalues()
+        return iter(self.material_keys.values()) if sys.version_info.major > 2 else self.material_keys.itervalues()
```

Comment:
```suggestion
        return iter(self.material_keys.values())
```

Reply:
yes but we are supporting ironpython

Reply:
I noticed Iron Python is 2.7 is meant to be supported, and values() returns a copy of the list in Python 2. 

If that matters there are some options [here](https://python-future.org/compatible_idioms.html#dictionaries). Which is a bit cleaner.

It would add a dependency on `future` module though, and from a search for `itervalues()` and `iteritems()` in the code base, there appears to be only one other place this happens, which does it in a similar way to my original commit.

Reply:
Very good point, thanks for thinking about IronPython.